charts/prometheus-adapter/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: prometheus-adapter
-version: 3.4.2
+version: 3.5.0
```

Lgtm but as it seams to be a Bugfix I would only raise the version to 3.4.3.

```suggestion
version: 3.4.3
```

---

Done